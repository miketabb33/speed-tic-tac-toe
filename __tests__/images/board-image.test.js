import renderer from 'react-test-renderer'
import BoardImage from '../../images/board-image'

var subject

const someImageSize = 100
const someBoardIndex = 2

beforeEach(()=> {
  subject = new BoardImage
})

test('x image rendering', () => {
  const rendering = renderer.create(
    subject.get(someImageSize, someBoardIndex)
  )
  
  const result = rendering.toJSON()
	expect(result).toMatchSnapshot()
})