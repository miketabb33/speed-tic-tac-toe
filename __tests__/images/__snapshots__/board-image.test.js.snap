// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`x image rendering 1`] = `
<div
  style={
    Object {
      "boxSizing": "border-box",
      "display": "inline-block",
      "margin": 0,
      "maxWidth": "100%",
      "overflow": "hidden",
      "position": "relative",
    }
  }
>
  <div
    style={
      Object {
        "boxSizing": "border-box",
        "display": "block",
        "maxWidth": "100%",
      }
    }
  >
    <img
      alt=""
      aria-hidden={true}
      src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTAwIiBoZWlnaHQ9IjEwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
      style={
        Object {
          "border": "none",
          "display": "block",
          "margin": 0,
          "maxWidth": "100%",
          "padding": 0,
        }
      }
    />
  </div>
  <img
    alt="board-2"
    data-nimg="intrinsic"
    decoding="async"
    draggable="false"
    id="board-2"
    src="/_next/image?url=%2Fboard.png&w=256&q=75"
    srcSet="/_next/image?url=%2Fboard.png&w=128&q=75 1x, /_next/image?url=%2Fboard.png&w=256&q=75 2x"
    style={
      Object {
        "border": "none",
        "bottom": 0,
        "boxSizing": "border-box",
        "display": "block",
        "height": 0,
        "left": 0,
        "margin": "auto",
        "maxHeight": "100%",
        "maxWidth": "100%",
        "minHeight": "100%",
        "minWidth": "100%",
        "objectFit": undefined,
        "objectPosition": undefined,
        "padding": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "width": 0,
      }
    }
  />
  <noscript>
    <img
      alt="board-2"
      data-nimg="intrinsic"
      decoding="async"
      draggable="false"
      id="board-2"
      loading="lazy"
      src="/_next/image?url=%2Fboard.png&w=256&q=75"
      srcSet="/_next/image?url=%2Fboard.png&w=128&q=75 1x, /_next/image?url=%2Fboard.png&w=256&q=75 2x"
      style={
        Object {
          "border": "none",
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": 0,
          "left": 0,
          "margin": "auto",
          "maxHeight": "100%",
          "maxWidth": "100%",
          "minHeight": "100%",
          "minWidth": "100%",
          "objectFit": undefined,
          "objectPosition": undefined,
          "padding": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 0,
        }
      }
    />
  </noscript>
</div>
`;
